commands/operator-sdk/cmd/build.go

```diff
@@ -144,14 +145,20 @@ func buildFunc(cmd *cobra.Command, args []string) {
 		cmdError.ExitWithError(cmdError.ExitBadArgs, fmt.Errorf("build command needs exactly 1 argument"))
 	}
 
+	cmdutil.MustInProjectRoot()
```

Comment:
This command breaks support for Ansible Operator because `config/config.yaml` doesn't exist in our project. Why doesn't this function look for the Dockerfile instead? It would verify we are in the project root and be agnostic to the type of operator.